packages/Icon/README.md

```diff
@@ -0,0 +1,31 @@
+# Icon
+
+## Description
+
+`<Icon>` component is a graphic to represent something else
+
+## Installation
+
+```
+yarn add @paprika/icon
+```
+
+or with npm:
+
+```
+npm install @paprika/icon
+```
+
+## Usage
+
+If you need a custom icon for paprika, you can use the `@paprika/icon`
+
+```jsx
+import AddIcon from "@paprika/icon";
```

Comment:
can we add a link to where the rest of the icons are like the folder of the filenames that we use to transform this icons?

Reply:
like here:
https://github.com/acl-services/paprika/tree/2964281695972de5c364a30acbc2121412842c16/packages/Icon/src/svg

Also adding that each filename will be transformed into PascalName style?